### PR TITLE
Batch database removal (421476)

### PR DESCRIPTION
**Pull request by KNyce71**
Per Ellery Murdock, all instances of PI Batch should be removed from interface guides. The following issues were addressed in this PR for Rockwell Factory Talk:

- **Introduction**  Removed PI Batch Database from Stores bullet point
- **Batch interface overview**: Reworked intro paragraph, deleted buffering paragraph. 
- **Batch interface configuration**: Deleted PI Batch database image
- **How interfaces process batch event data**: Removed variations of PIBatch
- **Recipe templates**: Removed PI Module Database from ModulePath setting
- **Logging and error messages**: Changed log name to Rockwell
- Updated TOC

### CODE REVIEW

File: content/installation-prerequisites/installation-prerequisites/logging-and-error-messages.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Specifies level of detail for logging as follows:
 
 **Log properties**
 
-You can create the Source in the Windows Event Viewer by using the batch interface's source/log name (**PIEMDVB**) before initially starting the interface. After starting the interface, the Windows Event Viewer creates the Source automatically.
+You can create the Source in the Windows Event Viewer by using the batch interface's source/log name (**PIFTBInt**) before initially starting the interface. After starting the interface, the Windows Event Viewer creates the Source automatically.
```
Comment (alan-izar) [inline content/installation-prerequisites/installation-prerequisites/logging-and-error-messages.md:49]: Is there a way to use a placeholder here for each interface? This is going to be common text across the interfaces so we'd only need to apply the specific name for each of the logs, unless that's too much work? If so, then this is correct.

Comment (KNyce71) [reply content/installation-prerequisites/installation-prerequisites/logging-and-error-messages.md:49]: Yes, in Framework I have set up a placeholder and have a list of the source log names hidden so I can reference them. When I added this topic thru the Framework and went in to modify it, I mistakenly put EMDVB's log name there. User error.

Comment (alan-izar) [reply content/installation-prerequisites/installation-prerequisites/logging-and-error-messages.md:49]: Got it. Thanks!

**Review by alan-izar — APPROVED** [review]

File: content/how-batch-interfaces-work/templates-for-mapping-data-source-events/recipe-templates.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ The interface includes a set of built-in default recipe templates that control t
     --|--
     **NAME** | (Required) Defines the convention used by the interface to assign names to procedures, unit procedures, etc. You can use the advanced parsing parameters to define this field. <br><br>Example: `abc_[PROCEDURE]` <br><br>If the procedure field of the incoming event contains `Test`, the resulting name is `abc_Test`.
     **BATCHID** | (Optional) Specifies the batch ID for the procedure or unit procedure.
-    **MODULEPATH** | (Optional) For unit procedures (level 2) or phase (level 4), specifies where the recipe resides in the PI Module Database or PI AF element hierarchy. In the Module Database, the path specifies the location of the PIUnit for the unit batch.
+    **MODULEPATH** | (Optional) For unit procedures (level 2) or phase (level 4), specifies where the recipe resides in the PI AF element hierarchy. In the Module Database, the path specifies the location of the PIUnit for the unit batch.
```
Comment (alan-izar) [inline content/how-batch-interfaces-work/templates-for-mapping-data-source-events/recipe-templates.md:21]: ~In the Module Database, the path specifies the location of the PIUnit for the unit batch.~

**Review by alan-izar — CHANGES_REQUESTED** [review]

**Review by alan-izar — APPROVED** [review]

File: content/introduction/introduction.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ PI Interface for Rockwell FactoryTalk Batch [!include[version](../includes/produ
   
 * **Converts** the data to PI tags and PI batch properties.
 
-* **Stores** the collected data as event frames and elements within a supported database: PI Batch Database or PI Asset Framework (PI AF) Database.
+* **Stores** the collected data as event frames and elements within a supported database (PI Asset Framework (PI AF) Database).
```
Comment (alan-izar) [inline content/introduction/introduction.md:13]: feels a little redundant. Maybe 

**Stores** the collected data as event frames and elements within a PI Asset Framework (PI AF) Database.

**Review by alan-izar — CHANGES_REQUESTED** [review]

**Review by alan-izar — APPROVED** [review]

File: content/how-batch-interfaces-work/templates-for-mapping-data-source-events/recipe-templates.md (hunk 2 above)
Comment (KNyce71) [reply content/how-batch-interfaces-work/templates-for-mapping-data-source-events/recipe-templates.md:21]: Done

File: content/introduction/introduction.md (hunk 3 above)
Comment (KNyce71) [reply content/introduction/introduction.md:13]: Done

**Pull request merged**